libs/component-sample/src/lib/feature/search-result-list-sample/search-result-list-sample-data.component.ts

```diff
@@ -10,6 +10,7 @@ export class SearchResultListSampleDataComponent {
   constructor(private change: ChangeDetectorRef) { }
 
   @Input() model;
+  @Input() dualDisplay;
```

Comment:
do we need this input? I think model information is enough.

Reply:
That is from an earlier commit, but it is not the current state of the branch:

https://github.com/GSA/sam-design-system/blob/29173-dual-record-display/libs/layout-sample/src/lib/ui/search-result-list-sample-data/search-result-list-sample-data.component.ts